trace-demo/README.md

```diff
@@ -0,0 +1,108 @@
+# Install OpenTelemetry Demo Application in Kyma
+
+## Overview
+
+These instructions will install the OpenTelemetry [demo application](https://github.com/open-telemetry/opentelemetry-demo) on a Kyma cluster using this [helm chart](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-demo).
+
+**CAUTION:** This instructions are under development and are relying on a prelimary state of the [Configurable Tracing](https://github.com/kyma-project/kyma/issues/11231) story. The goal is to have early with an E2E scenario for that feature. The instructions for now require to reconfigure the Istio and telemetry installation which is only possible using a custom Kyma OpenSource installation. In the final state, this reconfiguration will not be required anymore.
+
+## Prerequisites
+
+- Kyma OSS installed from main branch (`kyma deploy -s main`)
+- kubectl > 1.22.x
+- Helm 3.x
+
+### Preparation
+
+1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+
+    ```bash
+    export KYMA_NS="{namespace}"
+    ```
+1. If you don't have it created yet, now is the time to do so:
```

Comment:
```suggestion
1. If you don't have created a Namespace yet, do it now:
```

Does this really happen AFTER exporting the Namespace as a variable? Or before?

Reply:
Yes you can save the name to a variable first, it's a bit more convenient because you will use the same namespace name from the beginning and less errors can be made